documentation/manual/src/main/asciidoc/en-US/modules/getting-started.asciidoc

```diff
@@ -8,7 +8,7 @@ and retrieving an entity using Hibernate OGM.
 
 Before we can start, make sure you have the following tools configured:
 
-* Java JDK 6 or above
+* Java JDK 7 or above
 * Maven 3.x
```

Comment:
ok if you want but we never went so far. Normally JDK's don't change too often, and you'll need to double-check various docs and websites either way.